resources/lib/jellyfin/core/api.py

```diff
@@ -53,9 +53,6 @@ def _delete(self, handler, params=None):
 
     #################################################################################################
 
-    def try_server(self):
-        return self._get("System/Info/Public")
```

Comment:
Isn't used anywhere as far as I can tell, but it's still a change that is unrelated to WoL